src/Components/Quizz/Modal.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import './Modal.css';
+import PalpatineSon1 from './sound/palpatine.mp3'
+import YodaSon1 from './sound/Yoda.mp3'
+
+const modal = (props) => {
+
+    return (
+        <div className='ContentPicModalOff'>
+            <p className='contentModalP'>
+            Level 1
+            </p>
+            <div className="modal-wrapper"
+                style={{
+                    transform: props.show ? 'translateY(0vh)' : 'translateY(-100vh)',
+                    opacity: props.show ? '1' : '0'
+                }}>
+                 
+                {(props.show) && (props.resultat) 
+                    ?   <div>
+                            <img className='palpatine' src='https://www.stickpng.com/assets/images/580b57fbd9996e24bc43bdbe.png' alt='yoda'></img> 
+                            <audio autoPlay>
+                            <source src={YodaSon1} type="audio/mpeg"/>
+                            </audio> 
```

Comment:
Fix indentation

Reply:
It is an interesting way to play a sound. I don't know if there is any downside by doing this. I feel it's a bit of a hack to do it this way. I would have used something like this https://stackoverflow.com/questions/47686345/playing-sound-in-reactjs but I don't mind if you keep doing it this way.